third_party/mkl_dnn/BUILD

```diff
@@ -28,6 +28,27 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "build_with_onednn_v3",
+    define_values = {
+        "build_with_mkl": "true",
```

Comment:
Will `dnnl_sgemm` in Eigen contraction kernel still work with v3?
https://github.com/tensorflow/tensorflow/blob/924b8c18c2c1f61782a08c199461f0c086333686/tensorflow/tsl/framework/contraction/eigen_contraction_kernel.h#L171-L172

If so, we shouldn't require `build_with_mkl` here right? (I'm fine with removing the `build_with_mkl` match requirement later once we actually switch to v3. Just curious.)

Reply:
I just checked and `dnnl_sgemm` won't work with v3 (I see `eigen_contraction_kernel_no_mkl` object in my build directory). Can we remove this once we fully migrate to v3?

Reply:
Could you please clarify why it won't work with v3? I still see it in v3.1 code:
https://github.com/oneapi-src/oneDNN/blob/928065bd62372d2824900a86c5438bba407c98d1/src/common/gemm.cpp#L98

`eigen_contraction_kernel_no_mkl` probably just means missing some configs. We should be able to fix that in the BUILD file.

Reply:
Sorry for the confusion: oneDNN v3.x indeed supports `dnnl_sgemm` API.  

I meant to say that when building v3 using this this PR, `dnnl_sgemm` is most likely not using v3 since I see `eigen_contraction_kernel_no_mkl` object in my build directory. But as you mentioned above, it probably means that I missed adding some configs. 

Do you think we should fix this now so that `dnnl_sgemm` also works with v3?